docs: auth-ui views section (#12152)
supabase/supabase: apps/docs/pages/guides/auth/auth-helpers/auth-ui.mdx

import { createClient } from '@supabase/supabase-js'
import { Auth, ThemeSupa } from '@supabase/auth-ui-react'

const supabase = createClient('<INSERT PROJECT URL>', '<INSERT PROJECT ANON API KEY>')

const App = () => (
  <Auth
    supabaseClient={supabase}
    appearance={{ theme: ThemeSupa }}
    providers={['google', 'facebook', 'twitter']}
  />
)
```

### Supported Views

The Auth component is currently shipped with the following views:

- [Email Login](../auth-email)
- [Magic Link login](../auth-magic-link)
- [Social Login](../social-login)
- Update password
- Forgotten password

> We are planning on adding more views in the future which can be viewed in the [repo](https://github.com/supabase/auth-ui)

Reviewer: ```suggestion
We are planning on adding more views in the future.  Follow along on that [repo](https://github.com/supabase/auth-ui).
```
